lib/modsulator.rb

```diff
@@ -59,6 +59,8 @@ def convert_rows
     root = full_doc.root
 
     @rows.each do |row|
+      next if row['druid'].nil?
```

Comment:
Does this handle the case where blank?

Reply:
No, because at this point, row won't be blank, it will be loaded with all the fields, but if the input spreadsheet had an empty row all of the fields within the hash will be nil.